### ISSUE TITLE
any text training causes Segmentation fault 

### ISSUE THREAD

**Issue by melnarte** [pjreddie/darknet#1279]
Hello, train my own model on new text data causes the Segmentation fault:

`
artem@DESKTOP:~/darknet$ ./darknet rnn train cfg/rnn.train.cfg -file toto.txt
rnn
layer     filters    size              input                output
    0 RNN Layer: 256 inputs, 1024 outputs
                connected                             256  ->  1024
                connected                            1024  ->  1024
                connected                            1024  ->  1024
Unused field: 'hidden = 1024'
    1 RNN Layer: 1024 inputs, 1024 outputs
                connected                            1024  ->  1024
                connected                            1024  ->  1024
                connected                            1024  ->  1024
Unused field: 'hidden = 1024'
    2 RNN Layer: 1024 inputs, 1024 outputs
                connected                            1024  ->  1024
                connected                            1024  ->  1024
                connected                            1024  ->  1024
Unused field: 'hidden = 1024'
    3 connected                            1024  ->   256
    4 softmax                                         256
Learning Rate: 0.1, Momentum: 0.9, Decay: 0.001, Inputs: 256 73728 576
Segmentation fault (core dumped)
`
thanks for any ideas !

**Comment by JanKrcek** [comment]
Hello, I've been dealing with the same issue today and the problem for me was in the CUDNN code performing batch normalisation. Possibly a version mismatch issue.

My steps to get text rnn working:
1. Change folder path for backup weights in rnn.c
2. Try to rebuild darknet with CUDNN=0 (and CUDNN-HALF=0 in case you are using Alexey's version.)
3. Fry some eggs on your GPU, as it trains its novel writing skills.

Hope this helps, cheers!
